src/data/config.rs

```diff
@@ -15,15 +15,24 @@ pub const CONFIG: &str = "config.yaml";
 
 const ADDR: &str = "0.0.0.0";
 const PORT: &str = "3000";
+const DEFAULT_FORGE_API_PAGE_SIZE: u8 = 10;
```

Comment:
Also, I wonder if a u8 is enough. Perhaps we'd quickly hit a rate limit past 256... but some projects do release a lot a lot, and big queries might be useful for those that are just spinning up a quick rime whenever they upgrade their system. And then that makes me wonder if it's more efficient to just like go up to u32/u64size then, because even  65535 isn't actually a lot a lot... although, yea, in the lifetime of rime that would probably help like 3 people in total.

But I honestly don't know if there are enough, if any, performance costs involved with supporting those 3 people. Idk, wonder what you think about that.

Reply:
The forges we currently support either don't support paging (flakehub), or have a hard cap at 100 (github, gitlab, forgejo/gitea). So u8 is plenty. Also consider that pages can be *massive*. For example, a *single* release of [keyboardio/Chrysalis](https://github.com/keyboardio/chrysalis) is 18k. 100 of them is almost 2mb. That's a *lot* of data to download and process.

Especially for popular stuff, I don't think I'd like to download even half a meg of data for every request.

Reply:
I suppose it this need ever did arise, because we supported some hypothetical forge that had a limit past 100 then we'd probably be able to change this trivially. With the hard limit in place it obvs. makes no sense to increase this.